test/corpus/comments.txt

```diff
@@ -80,4 +93,25 @@ switch foo {
         (sequence_expression
           (expression_statement
             (number))))
-      (comment))))
+      (comment)))
+  (expression_statement
+    (if_expression
+      (value_identifier)
+      (block
+        (expression_statement
+          (value_identifier)))
+      (comment)
+      (else_clause
+        (block
+          (expression_statement
+            (value_identifier))))))
+  (comment)
+  (let_declaration
+    (let_binding
+      (value_identifier)
+      (number))
```

Comment:
🤔 Looks like there's a comment missing right here before the decorator.

Reply:
Yeah, I just realized that after I made this PR. I think the `_newline_and_comment` node discards the comment in this particular position.